PURL-TYPES.rst

```diff
@@ -253,6 +253,23 @@ github
       pkg:github/package-url/purl-spec@244fd47e07d1004
       pkg:github/package-url/purl-spec@244fd47e07d1004#everybody/loves/dogs
 
+githubactions
+------
+``githubactions`` for GitHub Actions:
+
+- The ``namespace`` is the user or organization. It is not case sensitive and
+  must be lowercased.
+- The ``name`` is the repository name. It is not case sensitive and must be
+  lowercased.
+- The ``subpath`` is used to point to a secondary action within a repository. It is not case sensitive and must be lowercased.
```

Comment:
It's arbitrary. codeql-action is a good example, because we have 4 different actions in that repository. That field just specifies the path to that specific action within the home repository. 
github/codeql-action/init
github/codeql-action/analyze
github/codeql-action/autobuild
github/codeql-action/upload-sarif

Reply:
I agree that the wording "secondary" is misleading, e.g. in https://github.com/github/codeql-action's case there is no GitHub Action at the root of the repository (no "primary" GitHub Action in this wording). We should somehow say that `subpath` is the directory containing the GitHub Action within the repository.